packages/python/plotly/plotly/graph_objs/layout/_annotation.py

```diff
@@ -319,6 +319,14 @@ def ay(self, val):
     @property
     def ayref(self):
         """
+        Indicates in what terms the tail of the annotation (ax,ay)  is
+        specified. If `pixel`, `ay` is a relative offset in pixels
+        from `y`. If set to a y axis id (e.g. "y" or "y2"), `ay` is
+        specified in the same terms as that axis. This is useful  for
+        trendline annotations which should continue to indicate  the
+        correct trend when zoomed.
+
+=======
```

Comment:
some sort of merge gremlin here, also this whole file contains nothing but whitespace changes :)

Reply:
Yes I think so, I have a line in my editor's config that removes trailing whitespace. Also it seems there's a botched merge in this file so I'm just checking out the file in master (it's generated anyways) :+1: